quickwit/quickwit-opentelemetry/src/otlp/traces.rs

```diff
@@ -91,19 +93,20 @@ doc_mapping:
     - name: span_fingerprint
       type: text
       tokenizer: raw
-    - name: span_start_timestamp_nanos
-      type: u64
-      indexed: false
-    - name: span_end_timestamp_nanos
-      type: u64
-      indexed: false
-    - name: span_start_timestamp_secs
+    - name: span_start_timestamp
       type: datetime
       input_formats: [unix_timestamp]
+      output_format: unix_timestamp_nanos
       indexed: false
       fast: true
       precision: seconds
```

Comment:
We may want to make range queries with milliseconds precisions. Does Jaeger make that possible?

Reply:
Yeah, we also get nanos on the Jaeger side, but the min/max timestamps are coming from the lookback selector with values such as `last x minutes`, or `last x hours`, so the end user does not have the opportunity to issue queries that would take advantage of the millisecond precision.

We do have a fast field in milliseconds for the span duration.

Reply:
Ok. I'm anticipating a bit, but I would expect to support milliseconds of precision in the future. Grafana typically sends date range queries at millisecond precision.